### PR TITLE
Add raw output data to admin execution models

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
https://github.com/flyteorg/flyteidl/pull/205 added raw output data as an output result to execution events. In turn, it makes sense to store this data inline for flyteadmin db models when output size is within a limit.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1327

## Follow-up issue
_NA_

### CODE REVIEW

**Review by EngHabu — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]
+1 but for posterity - don't confuse "Raw output metadata" with "Raw output data prefix"  the former is just the primitive output data that tasks return.  the latter is offloaded data like blobs.
